pkg/spec/security.go

```diff
@@ -123,6 +125,25 @@ func (c *SecurityConfig) ConfigureGenerator(g *generate.Generator, user *UserCon
 		return err
 	}
 
+	privCapRequired := []string{}
+
+	if !c.Privileged && len(c.CapRequired) > 0 {
```

Comment:
I think we should always set `CapRequired`. If it's not set via the CLI or by the image, we should use the default ones.

Reply:
We can talk about this, but this could be a problem.  I was thinking CapRequired was something that the user or the image had specified as the capabilies required to run the application.  As  opposed to the caplist, which is the "default capabilities + CapADD - CapDrop).  If we set CapRequired == CapDefault and only overrode it when the label was set, then we would not be able to tell when the user specified at capability outside of the "default" list of capabilities.

Reply:
Maybe we can have a quick bluejeans later? I want to show the `pkg/seccomp` package that we can extend for these needs.